CMakeLists.txt

```diff
@@ -53,3 +54,11 @@ if(BUILD_munich_graph_sim)
   execute_process(COMMAND cmake ".." WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/munich_graph_sim/build")
   execute_process(COMMAND cmake "--build" "." "-j${NUM_JOBS_BUILD}" WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/munich_graph_sim/build")
 endif()
+
+if(BUILD_abm_test_bs_simulation)
+  if(NOT EXISTS "${CMAKE_SOURCE_DIR}/ABM_TEST_BS_simulation/build")
+    execute_process(COMMAND mkdir "build/" WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/ABM_TEST_BS_simulation")
```

Comment:
This is not the name of your folder, therefore, the build is not successful. :) Maybe you could test it next time ;)